src/applications/edu-benefits/0994/pages/highTechIndustry.js

```diff
@@ -53,8 +56,7 @@ export const uiSchema = {
     },
     highTechnologyEmploymentType: {
       'ui:title': ' ',
-      'ui:description':
-        'Which area best describes your high-tech work experience? (Check all that apply.)',
+      'ui:description': highTechnologyEmploymentTypeDescription,
```

Comment:
This now results in two legends in the markup for one fieldset, though one is empty. I think that's ok for now, but I need to go and figure out what the right pattern is checkbox group labels and update the forms library to support this better.

Reply:
Concur, this was a workaround.